src/utils/service_worker.cr

```diff
@@ -91,8 +91,8 @@ module Mint
         .glob("**/*")
         .reject { |file| File.directory?(file) }
         .reduce(OpenSSL::Digest.new("SHA256")) do |digest, file|
-          digest.update File.read(file)
-        end.to_s
+        digest.update File.read(file)
+      end.to_s
```

Comment:
That seems wrong... might a formatter bug.

Reply:
@KevinSjoberg did you use Crystals formatter here?

Reply:
@gdotdesign yes, I ran `crystal tool format`.

Reply:
Just noticed I was a couple of versions behind. I was using `0.30.1`. Upgrading to `0.33.0` now.

Reply:
I've updated the pull request with the changes from running `crystal tool format` using the latest version of Crystal (0.33.0).